okhttp-testing-support/src/main/kotlin/okhttp3/OkHttpClientTestRule.kt

```diff
@@ -13,6 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@file:Suppress(
```

Comment:
I don’t think the compiler needed these ’cause of the _friends_ feature, but the IDE really wanted ’em